filters/diag/diag.go

```diff
@@ -135,6 +153,28 @@ func NewBackendBandwidth() filters.Spec { return &throttle{typ: backendBandwidth
 //
 func NewBackendChunks() filters.Spec { return &throttle{typ: backendChunks} }
 
+// NewRandomJitter creates a jitter for requests with random
```

Comment:
Should it be "uniform" instead of "random" to be along with "normal" and "gaussian"?

Reply:
We have `latency` and `backendLatency` that implement constant delay (IMO naming is not perfect - `latency` introduces delay on response path while `backendLatency` introduces delay on request path)

Should we add these as 
* `backendLatencyUniform`
* `backendLatencyNormal`
(and maybe add `latencyUniform` and `latencyNormal` later on) to stay consistent?

Reply:
great idea!

Reply:
done

Reply:
I think this was not resolved, will keep discussion open for now

Reply:
I would rather go for, but I am not sure if `jitter` is not the better name:
`responseLatencyUniform`
`responseLatencyNormal`

Maybe better `responseLatency` and deprecate `backendLatency`?

Reply:
internally discussed -> `normalRequestLatency(), normalResponseLatency(), uniformRequestLatency(), uniformResponseLatency()`